frontend/app/controllers/Bundle.scala

```diff
@@ -11,6 +11,50 @@ import views.support.{Asset, PageInfo}
 
 trait Bundle extends Controller {
 
+  private def landingMainImage () = {
+
+    val imgId = "d1a7088f8f2a367b0321528f081777c9b5618412/0_0_3578_2013"
+    val imgSizes = Seq(2000, 1000, 500)
+
+    ResponsiveImageGroup(
+      availableImages = ResponsiveImageGenerator(imgId, imgSizes)
+    )
```

Comment:
It can be a `val`!

```
private val landingMainImage = ResponsiveImageGroup(availableImages = 
  ResponsiveImageGenerator("d1a7088f8f2a367b0321528f081777c9b5618412/0_0_3578_2013", Seq(2000, 1000, 500)
)
```